### PR TITLE
Add Release notes for KubeDB v2024.11.18

### PR DESCRIPTION
**Pull request by tapojit047**

### CODE REVIEW

**Review by Evanraisul — DISMISSED** [review]
LGTM

File: content/post/kubedb-v2024.11.18/index.md [hunk 1]
```diff
@@ -0,0 +1,898 @@
...
+
+### Monitoring
+
+This release introduces an enhanced monitoring feature for KubeDB-managed Cassandra deployments by integrating Grafana dashboards. These dashboards provide comprehensive insights into various Cassandra-specific metrics, statuses, as well as visual representations of memory and CPU consumption. With this dashboard, users can effortlessly assess the overall health and performance of Cassandra, enabling more informed decision-making and efficient resource management.
+
+Have a look [here](https://github.com/ops-center/grafana-dashboards/tree/master/cassandra)  for a step-by-step guide on using the monitoring feature of Apache Cassandra.
+
+Here’s a preview of the Summary dashboard for Cassandra:
+
+![Cassandra Summary Dashboard](cassandra-summery.png)
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.11.18/index.md:72]: rename to `cassandra-summary` ?

File: content/post/kubedb-v2024.11.18/index.md [hunk 2]
```diff
@@ -0,0 +1,898 @@
...
+- zookeeper
+---
+
+We are thrilled to announce the release of **KubeDB v2024.11.18**. This release introduces several key features, including:
+
+- **TLS/SSL Support**: TLS/SSL support has been implemented for both Druid and Memcached, significantly improving security by enabling encrypted communication.
+
+- **OpsRequest Support**: Enhanced operational request capabilities for Druid, Memcached, Microsoft SQL Server, and Solr, providing greater management flexibility.
+
+- **Rotate Auth**: A new Ops Request named `RotateAuth` has been introduced. This feature enables users to rotate the credentials of the database enhancing overall security. It is initially added for `Druid`, `Kafka`, `MongoDB`, `Postgres`, and `Solr`.
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.11.18/index.md:50]: also added for elasticsearch

File: content/post/kubedb-v2024.11.18/index.md [hunk 3]
```diff
@@ -0,0 +1,898 @@
...
+        name: druid-cluster
+    updateVersion:
+        targetVersion: 30.0.1
+    timeout: 5m
+    apply: IfReady
+```
+
+**Reconfigure**
+
+```
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.11.18/index.md:159]:
```yaml

**Review by obaydullahmhs — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2024.11.18/index.md [hunk 4]
```diff
@@ -0,0 +1,898 @@
...
+kind: KafkaOpsRequest
+metadata:
+  name: kfops-rotate-auth
+  namespace: demo
+spec:
+  type: RotateAuth
+  databaseRef:
+    name: kafka-prod
+  ```
+If the secret is referenced, the operator will update the .spec.authSecret.name` with the new secret name. Here is the yaml,
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.11.18/index.md:245]: missed a backtick here. .spec.authSecret.name`

**Review by obaydullahmhs — CHANGES_REQUESTED** [review]

**Review by HiranmoyChowdhury — DISMISSED** [review]
LGTM!

**Review by sayedppqq — APPROVED** [review]
LGTM!

**Review by Neaj-Morshad-101 — APPROVED** [review]
LGTM!

File: content/post/kubedb-v2024.11.18/index.md [hunk 5]
```diff
@@ -0,0 +1,1269 @@
...
+### Updates for MSSQLServer CRD
+
+**Removed Fields**:
+
+- `spec.internalAuth`: The `spec.internalAuth` section has been removed. TLS/SSL configurations for internal endpoint authentication of SQL Server Availability Group replicas should now be defined under `spec.tls`.
+
+**Updated Fields**:
+
+- Leader Election Configuration:
+  - The field `spec.leaderElection` is now moved under `spec.topology.availabilityGroup.leaderElection`.
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.11.18/index.md:583]: Leader Election Configuration: The field `spec.leaderElection` has been moved to `spec.topology.availabilityGroup.leaderElection`.

File: content/post/kubedb-v2024.11.18/index.md [hunk 6]
```diff
@@ -0,0 +1,1269 @@
...
+          - kubedb
+    - alias: client
+      emailAddresses:
+        - abc@appscode.com
+      secretName: mssqlserver-client-cert
+      subject:
+        organizationalUnits:
+          - client
+        organizations:
+          - kubedb
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.11.18/index.md:610]: We can remove this part.

File: content/post/kubedb-v2024.11.18/index.md [hunk 7]
```diff
@@ -0,0 +1,1269 @@
...
+      heartbeatTick: 1
+      period: 300ms
+      transferLeadershipInterval: 1s
+      transferLeadershipTimeout: 1m0s
+  mode: AvailabilityGroup
+```
+
+**Additional Notes:**
+- Use `spec.tls` to configure all TLS/SSL-related settings, including endpoint authentication for Availability Group replicas.
+- For leader election, adjust parameters under `spec.topology.availabilityGroup.leaderElection` as needed.
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.11.18/index.md:635]: > Note: Use `spec.tls` to configure all TLS/SSL-related settings, including endpoint authentication for Availability Group replicas.

Remove it:
- For leader election, adjust parameters under `spec.topology.availabilityGroup.leaderElection` as needed.

File: content/post/kubedb-v2024.11.18/index.md [hunk 8]
```diff
@@ -0,0 +1,1269 @@
...
+        env:
+        - name: ACCEPT_EULA
+          value: "Y"
+        - name: MSSQL_PID
+          value: Enterprise
+```
+
+In this example, the SQL Server container will run the **Enterprise Edition** of SQL Server.
+
+For a complete list of environment variables and their usage, refer to the [official Microsoft documentation](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-2017).
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.11.18/index.md:673]: To know more about these environment variables and their usage,

**Review by Neaj-Morshad-101 — CHANGES_REQUESTED** [review]

**Review by Neaj-Morshad-101 — DISMISSED** [review]

**Review by pritamdas99 — DISMISSED** [review]
LGTM!

**Review by raihankhan — APPROVED** [review]
LGTM 👍🏻

**Review by souravbiswassanto — DISMISSED** [review]

**Review by ArnobKumarSaha — DISMISSED** [review]

**Pull request merged**